lib/discourse.php

```diff
@@ -466,17 +464,13 @@ function sync_to_discourse_work( $postid, $title, $raw ) {
       $url =  $options['url'] .'/posts';
 
       // use key 'http' even if you send the request to https://...
-      $soptions = array(
-        'http' => array(
-          'ignore_errors' => true,
-          'method'  => 'POST',
-          'content' => http_build_query( $data ),
-          'header' => "Content-Type: application/x-www-form-urlencoded\r\n"
-        )
+      $post_options = array(
+        'timeout'     => 30,
```

Comment:
remove this extra spacing, then squash these commits into one and this will be ready to merge